spec/dummy/db/schema.rb

```diff
@@ -332,4 +333,9 @@
     t.index ["uid"], name: "index_users_on_uid", unique: true
   end
 
+  add_foreign_key "chapters", "topics"
```

Comment:
wtf

Reply:
That’s fine, we recently added foreign keys for contents in mumuki/mumuki-domain#21.